plugins/frontend/backstage-plugin-github-pull-requests/src/components/PullRequestsListView/PullRequestsListView.tsx

```diff
@@ -66,6 +67,45 @@ const useStyles = makeStyles(theme => ({
   },
 }));
 
+export const SkeletonPullRequestItem = () => {
+  const classes = useStyles();
+  return (
+    <Grid container item spacing={0} className={classes.pullRequestRow} xs={12}>
+      <Grid item xs="auto">
+        <Skeleton
+          variant="circle"
+          width={18}
+          height={18}
+          style={{ marginTop: '3px' }}
+        />
+      </Grid>
+      <Grid item xs={10} className={classes.middleColumn}>
+        <Typography variant="body1">
+          <Skeleton variant="text" />
+        </Typography>
+        <Typography variant="caption">
+          <Skeleton variant="text" width={130} />
+        </Typography>
+      </Grid>
+      <Grid
+        item
+        spacing={1}
+        xs={1}
+        style={{ flexShrink: 0, marginLeft: 'auto' }}
+        className={classes.secondaryText}
+      >
+        <Box display="flex" justifyContent="flex-end">
+          <Skeleton
+            variant="circle"
+            width={18}
+            height={18}
```

Comment:
How do you determine size which is optimal?

Reply:
trial and error :D I think it looks ok with the 18px